data/cod.md

```diff
@@ -1,52 +1,28 @@
-### 1. Purpose
+## Code de conduite pour la conférence
 
-A primary goal of [GDG Lviv](http://lviv.gdg.org.ua/) is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof). 
-GDG Lviv prides itself on being an open, respectful, and inclusive community. That means the [jerky behavior](https://meta.wikimedia.org/wiki/Don%27t_be_a_jerk) isn’t allowed at GDG meetings.
+Tous les participants, conférenciers, sponsors et volontaires pour notre conférence doivent accepter le code de conduite suivant. Les organisateurs s'attacheront à faire respecter ce code durant l'événement. Nous attendons de la part de chaque participant une coopération pour assurer un environnement sain pour tous.
 
-This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
-We invite all those who participate in Google Developers Group to help us create safe and positive experiences for everyone.
+**En bref, ne soyez pas un imbécile.**
```

Comment:
Hum ? Bof non ?

Reply:
C'est le standard, c'est le "code of conduct" le plus utilisé (français et anglais), donc c'est validé par la communauté ;-)